.github/workflows/build-image-pr.yml

```diff
@@ -31,3 +31,24 @@ jobs:
         env:
           VERSION: ${{ steps.tags.outputs.tag }}
         run: ./scripts/build_deploy.sh
+      - name: Start Kind Cluster
+        uses: helm/kind-action@v1.9.0
+      - name: Load Local Registry Test Image
+        env:
+          IMG: "quay.io/opendatahub/model-registry:${{ steps.tags.outputs.tag }}"
```

Comment:
Either in this PR or in a separate one we'll need to replace references to quay with dockerhub instead. I think this action doesn't push/pull images from quay, but it could be confusing to a new reader.

Reply:
Or better yet, replace with generic variables like `DOCKER_REPO`.

Reply:
This PR focus on Kind testing.

Indeed that can/will be done in separate PR

Reply:
>  I think this action doesn't push/pull images from quay

can confirm, here:

https://github.com/kubeflow/model-registry/blob/962c78df14057e29952d126fd60faf73813be040/.github/workflows/build-image-pr.yml#L15

once Kind is in-place as it was pre-migration, we can use parametric image name also here, and would be a great non-regression testing that image creation and publishing (to dockerhub).